bool map_node_rdy